src/main/java/io/swagger/codegen/v3/generators/handlebars/java/JavaHelper.java

```diff
@@ -79,20 +79,26 @@ public CharSequence getModelImports(Map<String, Object> templateData, Options op
         boolean withXml = Boolean.valueOf(String.valueOf(templateData.get("withXml")));
         boolean parcelableModel = Boolean.valueOf(String.valueOf(templateData.get("parcelableModel")));
         boolean useBeanValidation = Boolean.valueOf(String.valueOf(templateData.get("useBeanValidation")));
+        boolean jakarta = Boolean.valueOf(String.valueOf(templateData.get("jakarta")));
         if (serializableMode) {
             builder.append("import java.io.Serializable;\n");
         }
         if (jackson && withXml) {
             builder.append("import com.fasterxml.jackson.dataformat.xml.annotation.*;\n");
         }
-        if (withXml) {
-            builder.append("import javax.xml.bind.annotation.*;\n");
+        if (withXml && jakarta) {
+            builder.append("import jakarta.xml.bind.annotation.*;\n");
+        } else if (withXml) {
+            builder.append("import jakarta.xml.bind.annotation.*;\n");
```

Comment:
Looks like "import javax.xml.bind.annotation.*;\n" is expected here

Reply:
You're right
I'll try to push fix today